public/script.js

```diff
@@ -5,8 +5,8 @@ const fqdn = window.location.host;
 
 const DEFAULT_EXPIRY = 60 * 60; // one hour
 const PASSPHRASE_CHARSET =
-  "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz~!@#$%^&*()?";
-const PASSPHRASE_LEN = 64;
+  "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz@#$%^&?";
+const PASSPHRASE_LEN = 40;
```

Comment:
I pulled from latest master upstream before creating the wrong-passphrase branch. For some reason, this update did not get included. Therefore, I'm adding this back ...

Reply:
Let's not do that, instead pls re-pull master and rebase your feature branch. I suspect that when you pulled master my PR had not been merged yet?

Make sure to undo this change before rebase or else you will have merge conflict

Reply:
Your change doesn't even depend on mine so theoretically you wouldn't care if you had my change or not. Once your PR is merged into master everything should be up to date in master

Reply:
that makes sense. Okay, I'll do a rebase